docker/docker_client.go

```diff
@@ -388,7 +388,13 @@ func SearchRegistry(ctx context.Context, sys *types.SystemContext, registry, ima
 	}
 
 	logrus.Debugf("trying to talk to v2 search endpoint")
-	resp, err := client.makeRequest(ctx, "GET", "/v2/_catalog", nil, nil, v2Auth, nil)
+	u := url.URL{
+		Path: "/v2/_catalog",
+	}
+	q := u.Query()
+	q.Set("n", strconv.Itoa(limit))
```

Comment:
Does limit get set to 100 if not specified by the user?

Reply:
I think Podman search can limit this to its default number of result 25. https://github.com/containers/libpod/blob/3eb0ad04a8b1d56866a16f1428bb8019927ccfa3/libpod/image/search.go#L136-L139